exchanges/kucoin/kucoin.go

```diff
@@ -164,11 +134,14 @@ func (e *Exchange) GetOrderbook(ctx context.Context, symbol string) (*Orderbook,
 	params := url.Values{}
 	params.Set("symbol", symbol)
 	var o *orderbookResponse
-	err := e.SendAuthHTTPRequest(ctx, exchange.RestSpot, fullOrderbookEPL, http.MethodGet, common.EncodeURLValues("/v3/market/orderbook/level2", params), nil, &o)
-	if err != nil {
+	if err := e.SendAuthHTTPRequest(ctx, exchange.RestSpot, fullOrderbookEPL, http.MethodGet, common.EncodeURLValues("/v3/market/orderbook/level2", params), nil, &o); err != nil {
 		return nil, err
 	}
-	return constructOrderbook(o)
+	return unifySpotOrderbook(o), nil
+}
+
+func unifySpotOrderbook(o *orderbookResponse) *Orderbook {
```

Comment:
🤷‍♂️ Super unsure why we should have this function _today_ 🙂 
If we'll want it in the future, we add it in the future.